lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -145,7 +145,6 @@ EOS
 # Please put the most recent first.
 my $sw_legacy_java = Genome::Config::get('sw_legacy_java');
 my @PICARD_VERSIONS = (
-    '1.123' => '/gscmnt/sata132/techd/solexa/jwalker/lib/picard-tools-1.123',
```

Comment:
It was debian packaged by Ian a while back.  The hash version should not be necessary.